### PR TITLE
chore: update eslint

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

Updated eslint packages to the latest versions and fixed `'err' is defined but never used  unused-imports/no-unused-vars` error

### CODE REVIEW

**Comment by shoom3301** [comment]
@alfetopito the IDEA integration with eslint works well to me now

File: package.json
Comment (alfetopito) [inline package.json:1]: The fix was simply to update everything? 🤦

File: apps/explorer/src/utils/miscellaneous.ts [hunk 1]
```diff
@@ -18,13 +16,10 @@ export function assertNonNull<T>(val: T, message: string): asserts val is NonNul
   }
 }
 
-// eslint-disable-next-line
 function noop(..._args: any[]): void {}
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (alfetopito) [inline apps/explorer/src/utils/miscellaneous.ts:24]: Is the `no-explicit-any` rule disabled?

**Review by alfetopito — APPROVED** [review]

File: apps/explorer/src/utils/miscellaneous.ts (hunk 1 above)
Comment (shoom3301) [reply apps/explorer/src/utils/miscellaneous.ts:24]: It was disabled even before.
For example: https://github.com/cowprotocol/cowswap/blob/e4a89e8775324651a1fa85c1ba4ae785e33d8a1f/apps/explorer/src/components/orders/NumbersBreakdown/index.tsx#L48 there is no error

**Pull request merged**